### PR TITLE
LPS-66950 build-lang

### PR DESCRIPTION
**Pull request by pavel-savinov**

### CODE REVIEW

**Comment by pavel-savinov** [comment]
ci:retest

**Comment by JorgeFerrer** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: portal-impl/src/com/liferay/portlet/asset/service/permission/AssetPermission.java [hunk 1]
```diff
@@ -30,20 +33,29 @@
```
Comment (JorgeFerrer) [inline portal-impl/src/com/liferay/portlet/asset/service/permission/AssetPermission.java]: This class looks good, but I think it should be left as deprecated and two new classes should be created: AssetCategoriesPermission and AssetTagsPermission (note that categories and tags are plural).

That way we keep the common pattern that each resource has its class. What do you think?

File: portal-impl/src/com/liferay/portal/upgrade/v7_0_3/UpgradePermission.java [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+			runSQL(sb.toString());
+		}
+		catch (Exception e) {
+			throw new UpgradeException(e);
+		}
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
```
Comment (JorgeFerrer) [inline portal-impl/src/com/liferay/portal/upgrade/v7_0_3/UpgradePermission.java]: I didn't have time to do an in-depth review of this class looking for potential bugs, but conceptually it makes sense to me.

**Pull request closed without merging**